server/belga/image.py

```diff
@@ -212,6 +212,93 @@ def format_list_item(self, data):
         }
 
 
+class Belga360ArchiveSearchProvider(superdesk.SearchProvider):
+
+    GUID_PREFIX = 'urn:belga.be:360archive:'
+
+    label = 'Belga 360 Archive'
+    base_url = 'http://mules.staging.belga.be:48080/belga360-ws/'
+    search_endpoint = 'archivenewsobjects'
+    items_field = 'newsObjects'
+    count_field = 'nrNewsObjects'
+
+    def __init__(self, provider, **kwargs):
+        super().__init__(provider, **kwargs)
+        self.session = requests.Session()
+        self._id_token = None
+        self._auth_token = None
+
+    def url(self, resource):
+        return urljoin(self.base_url, resource.lstrip('/'))
+
+    def find(self, query, params=None):
+        page_size = query.get('size', 25)
+        api_params = {
+            'start': query.get('from', 0) // page_size,
```

Comment:
Is there any chance that `page_size` will be `0` ? :)
BTW. Do we actually need `// page_size` here? Is it now enough to use `from` and `size` directly from a query?

Reply:
After rechecked, `page_size` is not necessary, we mistook `start` = start page